README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/sul-dlss/dor-workflow-service.svg?branch=master)](https://travis-ci.org/sul-dlss/dor-workflow-service)
```

Comment:
That's just how it's done. There's no fancy integration between github and travis to get any kind of contextual badge or anything like that. `master` is a pretty good default, too -- if you know `master` is good, and your branch isn't, it's probably your branch that broke something.